File: app/src/main/java/com/byd/dashcast/model/AppShortcut.java
package com.byd.dashcast.model;

import android.graphics.drawable.Drawable;

public class AppShortcut {
    public final String id;
    public final String label;
    public final Drawable icon;

    public AppShortcut(String id, String label, Drawable icon) {
        this.id = id;
        this.label = label;
        this.icon = icon;
    }
}

File: app/src/main/java/com/byd/dashcast/model/AppShortcut.kt
package com.byd.dashcast.model

import android.graphics.drawable.Drawable

// @JvmField keeps direct field access (shortcut.id) working from the
// remaining Java call sites during the Kotlin migration.
class AppShortcut(
    @JvmField val id: String,
    @JvmField val label: String,
    @JvmField val icon: Drawable?
)
